daml-script/test/src/test-utils/com/daml/lf/engine/script/test/AbstractFuncIT.scala

```diff
@@ -428,8 +448,10 @@ abstract class AbstractFuncIT
         }
       }
     }
+    // TODO https://github.com/digital-asset/daml/issues/15882
```

Comment:
so sandbox on x supported consortium parties?

Reply:
I am not completely sure.

Reply:
It is failing with Canton, taking into account @nickchapman-da is refactoring current implementation, I do not think is worth to spend time with it.

Reply:
wfm, I was just surprised that sandbox on x even supported it.